documentation/dsls/DSL:-Ash.Reactor.md

```diff
@@ -226,6 +226,201 @@ Target: `Ash.Reactor.Dsl.Action`
 
 
 
+## reactor.bulk_create
+```elixir
+bulk_create name, resource, action \\ nil
+```
+
+
+Declares a step which will call a create action on a resource with a collection of inputs.
+
+> ### Check the docs! {: .warning}
+>
+> Make sure to thoroughly read and understand the documentation in `Ash.bulk_create/4` before using. Read each option and note the default values. By default, bulk creates don't return records or errors, and don't emit notifications.
+
+Caveats/differences from `Ash.bulk_create/4`:
+
+1. `max_concurrency` specifies the number of tasks that Ash will start to process batches, and has no effect on Reactor concurrency targets.  It's could be possible to create a very large number of processes if a number of steps are running bulk actions with a high degree of concurrency.
```

Comment:
I wonder if Ash should support being given an async limiter like reactor uses? Would be a pretty powerful concept.

Reply:
Yeah, we could definitely take the concurrency tracker out of reactor and repurpose it.